rust/basic_bitcoin/src/basic_bitcoin/basic_bitcoin.did

```diff
@@ -0,0 +1,37 @@
+type satoshi = nat64;
+type millisatoshi_per_byte = nat64;
+type bitcoin_address = text;
+type block_hash = blob;
+
+type outpoint = record {
+  txid : blob;
+  vout : nat32
+};
+
+type utxo = record {
+  outpoint: outpoint;
+  value: satoshi;
+  height: nat32;
+};
+
+type get_utxos_response = record {
+  utxos: vec utxo;
+  tip_block_hash: block_hash;
+  tip_height: nat32;
+  next_page: opt blob;
+};
+
+service : {
+    "get_p2pkh_address": () -> (bitcoin_address);
+
+    "get_balance": (address: bitcoin_address) -> (satoshi);
+
+    "get_utxos": (bitcoin_address) -> (get_utxos_response);
+
+    "get_current_fee_percentiles": () -> (vec millisatoshi_per_byte);
+
+    "send": (record {
+      destination_address: bitcoin_address;
+      amount_in_satoshi: satoshi;
```

Comment:
Maybe you answered this question and I missed it but wouldn't it be fine to just call it `amount` given that the unit is `satoshi`?
I'm not insisting on changing this but I'm just wondering if we gain much by calling it `amount_in_satoshi`.